wolox-books/src/app/screens/auth/bookdetail/bookdetail.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container-detail" *ngIf="(book | async)">
```

Comment:
https://blog.angularindepth.com/angular-question-rxjs-subscribe-vs-async-pipe-in-component-templates-c956c8c0c794

observables should end with an "$"
no need for specifying async on all the properties. you can do: *ngIf="book$ | async as book"